account_invoice_force_number/tests/test_account_force_number.py

```diff
@@ -4,49 +4,43 @@
 
 
 from odoo.exceptions import UserError
-from odoo.tests.common import TransactionCase
 
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+class TestAccountForceNumber(AccountTestInvoicingCommon):
+    def create_invoice(self, move_name=None):
+        partner_id = self.env.ref("base.res_partner_12")
```

Comment:
Even though the invoice creation doesn't raise any error, I'd prefer to get assignment compliant with the variable name:
`partner_id = self.env.ref("base.res_partner_12").id`

Reply:
No problem, now it's correct :ok_hand: